projects/addon-doc/src/components/example/example.component.ts

```diff
@@ -67,9 +73,8 @@ export class TuiDocExampleComponent {
         return this.activeItem === this.defaultTab;
     }
 
-    @tuiPure
-    private getTabs(content: Record<string, string>): readonly string[] {
-        return [this.defaultTab, ...Object.keys(content)];
+    get componentName(): string {
+        return this.windowRef.location.pathname.slice(1);
```

Comment:
Maybe it's better to add as input? After all example is a pretty generic component and can be used anywhere, independent of the router.